shell/edit/workload/Upgrading.vue

```diff
@@ -221,7 +231,7 @@ export default {
         />
       </div>
     </div>
-    <div v-if="isStatefulSet" class="row mb-20">
+    <div v-if="isStatefulSet&& !noDeploymentSpec" class="row mb-20">
```

Comment:
missing a space here...
`<div v-if="isStatefulSet&& !noDeploymentSpec" class="row mb-20">`
to 
`<div v-if="isStatefulSet && !noDeploymentSpec" class="row mb-20">`